src/Create.js

```diff
@@ -1,10 +1,85 @@
 import React, { Component } from 'react';
+import { getTypes, createDessert } from './fetch-utils.js';
+import classNames from 'classnames';
 
 // const URL = 'https://aqueous-coast-92629.herokuapp.com/desserts';
 
 class Create extends Component {
+    state = {
+            name: '',
+            icing: '',
+            type_id: 0,
+            message: '',
+            error: false,
+         };
+         componentDidMount = async () => {
+            const types = await getTypes();
+            this.setState({ types });
+        };
+
+        getDessertId = () => {
+            const dessertObject = this.state.desserts.find(
+                (dessert)=>dessert.name === this.state.network);
+            return dessertObject.id;
```

Comment:
Remember, if you're going to follow along with the demo code, you should be using as a reference and not copying it over into your code. You need to read through it and apply it to your own data structure, in which case you wouldn't be using `network` since that isn't applicable to your specific data set.